content/en/storage-provider/filecoin-economics/fil-collateral/index.md

```diff
@@ -19,11 +19,11 @@ As a storage provider on the network, you will have to create FIL wallets, and a
 
 ## Types of collateral
 
-To satisfy the multiple needs for collateral in a way that is minimally burdensome to storage providers, Filecoin includes three different collateral mechanisms: initial pledge collateral, block rewards as collateral, and storage deal provider collateral. The first is an initial commitment of Fil that a miner must provide with each sector. The second is a mechanism to reduce the initial token commitment by vesting block rewards over time. The third aligns incentives between storage provider and client, and can allow miners to differentiate themselves in the market. <!--TODO NOBLOCK STEF BOB needs more explanation or linking? is the second basically re-investing earned blocks to give greater future capacity?-->
+To satisfy the multiple needs for collateral in a way that is minimally burdensome to storage providers, Filecoin includes three different collateral mechanisms: initial pledge collateral, block rewards as collateral, and storage deal provider collateral. The first is an initial commitment of Fil that a miner must provide with each sector. The second is a mechanism to reduce the initial token commitment by vesting block rewards over time. The third aligns incentives between storage provider and client, and can allow miners to differentiate themselves in the market.
 
 For more detailed information about how collateral requirements are calculated, see the [miner collateral section in the Filecoin spec](https://spec.filecoin.io/systems/filecoin_mining/miner_collaterals/).
 
-When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), or when you take your storage offline and thereby break the rules of the storage deal, you will get penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited <!--TODO NOBLOCK STEF BOB where does it go? who gets it? Does it all go? -->and your storage power is reduced.
+When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), or when you take your storage offline and thereby break the rules of the storage deal, you will get penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited to the f099 address from your locked or available rewards and your storage power is reduced. The f099 address is the address in which all burned FIL go to.
```

Comment:
```suggestion
When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), storage is taken offline, or any storage deal rules are broken, the provider is penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited to the `f099` address from your locked or available rewards and your storage power is reduced. The `f099` address is the address where all burned FIL goes.
```

Reply:
@johnnymatthews should we use "Fil" or "FIL"?

Reply:
FIL.